README.md

```diff
@@ -11,6 +11,14 @@ gem 'lightningff', require: 'lightning'
 ```
 and run `bundle install`. _Note: You might need to run `bundle update` to resolve any incompatible issues with rails._
 
+#### Script Installation
+Run the following script to install the entire feature flag functionality without manual setup for your models (i.e. User, Workspace)
+```ruby
+rake lightning:install User Workspace
```

Comment:
Let's also mention that they can simply run `rails lightning:install`, and then perform those steps manually?

Reply:
btw, `rake` and `rails` are interchangeable commands — both work the same way.

Reply:
Updated it.